tests/testthat/test-geom-smooth.R

```diff
@@ -0,0 +1,13 @@
+context("geom_smooth")
+
+test_that("Data is setup as in geom_line", {
+  df <- data.frame(x = c(1, 5, 2, 3, 4), y = 1:5)
+
+  ps <- ggplot(df, aes(x, y))+
+    geom_smooth(stat = "identity", se = FALSE)
+
+  pl <- ggplot(df, aes(x, y))+
+    geom_line()
+
+  expect_equal(layer_data(ps)[c("x", "y")], layer_data(pl)[c("x", "y")])
```

Comment:
Rather than comparing to the output of another geom, the test would be simpler if you just compared to the correct output: 1:5.